backend/src/api/boardmember/content-types/boardmember/schema.json

```diff
@@ -0,0 +1,42 @@
+{
+    "kind": "collectionType",
+    "collectionName": "boardmembers",
+    "info": {
+        "singularName": "boardmember",
+        "pluralName": "boardmembers",
+        "displayName": "Bestuurslid",
+        "description": ""
+    },
+    "options": {
+        "draftAndPublish": true
+    },
+    "pluginOptions": {},
+    "attributes": {
+        "name": {
+            "type": "string",
+            "required": true
+        },
+        "role": {
+            "type": "string",
+            "required": true
+        },
+        "email": {
+            "type": "email",
+            "required": true
+        },
+        "extra": {
+            "type": "richtext"
+        },
+        "photo": {
+            "type": "media",
+            "multiple": false,
+            "required": true,
+            "allowedTypes": ["images"]
+        },
```

Comment:
Ik denk dat voor het gemak we deze velden beter optioneel kunnen maken. Bijv als iemand nog geen foto heeft opgestuurd naar degene die de bestuurders invult, is het handiger als die tijdelijk nog niet ingevuld is

Reply:
Alleen naam en volgorde is required. En een extra if toegevoegd in de template zodat mail icoontje en image, niet worden weergegeven als ze leeg zijn.